wallets/provider-clover/src/actions/solana.ts

```diff
@@ -0,0 +1,65 @@
+import type {
+  Subscriber,
+  SubscriberCleanUp,
+} from '@rango-dev/wallets-core/hub';
+
+import {
+  CAIP_NAMESPACE,
+  CAIP_SOLANA_CHAIN_ID,
+  type SolanaActions,
+} from '@rango-dev/wallets-core/namespaces/solana';
+import { AccountId } from 'caip';
+
+import { evmClover, solanaClover } from '../utils.js';
+
+export function changeAccountSubscriberAction(): [
```

Comment:
Add a note why we are defining a custom action here and what is the limation.

Reply:
Thanks for mentioning this, I've added the necessary explanations.